pkg/system/phase4_configuring.go

```diff
@@ -1401,3 +1405,43 @@ func (r *Reconciler) ReconcileNamespaceStores(namespaceResources []nb.NamespaceR
 	}
 	return nil
 }
+
+// ReconcileSystemConfigMap restarts the core and endpoint pods to recive updates from config map
+func (r *Reconciler) ReconcileSystemConfigMap() error {
+	sha256Hex := util.GetCmDataHash(r.CoreAppConfig.Data)
+
+	if (r.DeploymentEndpoint.ObjectMeta.Annotations["ConfigMapHash"] != sha256Hex) {
+		epPodList := &corev1.PodList{}
+		epPodSelector, _ := labels.Parse("noobaa-s3=" + r.Request.Name)
+		util.KubeList(epPodList, &client.ListOptions{Namespace: options.Namespace, LabelSelector: epPodSelector})
+		r.DeploymentEndpoint.ObjectMeta.Annotations["ConfigMapHash"] = sha256Hex
+
+		if !util.KubeUpdate(r.DeploymentEndpoint) {
+			r.Logger.Info("NooBaa %q failed to update Deployment Endpoint Config Map Hash", options.SystemName)
```

Comment:
should be r.Logger.Infof
```suggestion
			r.Logger.Infof("NooBaa %q failed to update Deployment Endpoint Config Map Hash", options.SystemName)
```